docs/quick-start.md

```diff
@@ -17,105 +18,156 @@ information at [planner usage](planner-usage.md).
+-   You can use the Docker flag `-rm` for cleaning up the container (recommended)
 
-The Docker image for Fast Downward is installed on your machine as a
-side-effect of the command.
+Assume you want to solve a planning task from the [Fast Downward benchmarks](https://github.com/aibasel/downward-benchmarks) using A* search with the LM-cut heuristic. To do so you can run the following:
 
-## Using a Vagrant machine
+    sudo docker run --rm -v <path-to-benchmarks>:/benchmarks aibasel/downward /benchmarks/<domain.pddl> /benchmarks/<task.pddl> --search "astar(lmcut())"
 
-We assume that
+Here `<path-to-benchmarks>` refers to your local directory containing your domain and task file.
 
--   [Vagrant](https://www.vagrantup.com/) is installed on
-    your machine
--   The current directory contains the Fast Downward `Vagrantfile` for
-    the desired [release](https://www.fast-downward.org/releases), the PDDL files
-    `domain.pddl` and `problem.pddl` for the planning task you want
-    to solve.
--   The !SoPlex LP solver is included automatically. If you want to also
-    use the CPLEX LP solver within the planner, its installer file must
-    be present in the directory `/path/to/lp/installers`. As of Fast
-    Downward 23.06, you will need CPLEX 22.1.1 (installer filename
-    `cplex_studio2211.linux-x86-64.bin`).
--   You want to create your Vagrant VM as subdirectory
-    `my-fast-downward-vm` of the current directory. The subdirectory
-    does not exist yet.
+Note:
 
-Create and provision your virtual machine as follows:
+-   The Docker flag `-v` mounts the local directory `<path-to-benchmarks>` of your host 
+    machine under the container directory `/benchmarks`, which is the
+    place where the containerised planner looks for the problem.
+-   `<path-to-benchmarks>` must be an absolute path; relative paths are only supported as of Docker Engine version 23.
+
+See more planner options at [planner usage](planner-usage.md).
+
+## Vagrant
+
+### Vagrant Installation
+To install Vagrant on your machine follow the steps provided on the [Vagrant page](https://developer.hashicorp.com/vagrant/install).
+For Vagrant to work you also need a "provider" such as VirtualBox, VMware, Hyper-V.
+
+Note:
+
+- Not all provider versions and Vagrant versions are compatible!
+- We recommend using Vagrant 2.4.1 and [VirtualBox 7.0.20](https://www.virtualbox.org/wiki/Download_Old_Builds_7_0).
+
+To create your virtual machine using Vagrant create a new directory `<my-fast-downward-vm>` containing only the Fast Downward Vagrantfile for
+the desired [release](https://www.fast-downward.org/latest/releases). 
+
+If you want to use LP solvers follow the steps under [LP Solvers](#lp_solvers).
+
+Otherwise, you can set up your virtual machine directly from the `<my-fast-downward-vm>` directory using:
+    
+    vagrant up
+
+#### LP solvers
+
+The !SoPlex LP solver is included automatically. If you want to also
```

Comment:
The ! in !SoplexSolver should be removed (left-over from the old MoinMoin wiki).